### PR TITLE
Release/v0.9.0

### PR DESCRIPTION
**Pull request by spencerkclark**

### CODE REVIEW

**Review by nbren12 — APPROVED** [review]
lgtm.

File: fv3config/__init__.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 __author__ = """Vulcan Technologies LLC"""
 __email__ = "jeremym@vulcan.com"
```
Comment (spencerkclark) [inline fv3config/__init__.py:32]: While I'm here, should I update this metadata?

File: setup.py [hunk 2]
```diff
@@ -62,6 +62,6 @@
     test_suite="tests",
     tests_require=test_requirements,
     url="https://github.com/VulcanClimateModeling/fv3config",
```
Comment (spencerkclark) [inline setup.py:64]: Here too?

Comment (nbren12) [reply setup.py:64]: although that will be changed by end of summer

**Comment by spencerkclark** [comment]
Should I replace Vulcan in these other places too?
```
$ grep -R Vulcan * 
LICENSE:Copyright (c) 2019, Vulcan Technologies LLC
README.rst:.. image:: https://circleci.com/gh/VulcanClimateModeling/fv3config.svg?style=svg
README.rst:   :target: https://circleci.com/gh/VulcanClimateModeling/fv3config
docs/installation.rst:    $ git clone git://github.com/VulcanClimateModeling/fv3config
docs/installation.rst:.. _Github repo: https://github.com/VulcanClimateModeling/fv3config
docs/conf.py:copyright = u"2019, Vulcan Technologies, LLC"
docs/conf.py:author = u"Vulcan Technologies, LLC"
docs/conf.py:        u"Vulcan Technologies, LLC",
```

**Pull request merged**